File: TYPESCRIPT_PRACTICAL_EXAM/question_1.js
"use strict";
// 1) Write a TypeScript program that checks if a given
// number is odd or even.
let check_even_odd = (num) => {
    if (num % 2 == 0) {
        return "even";
    }
    else {
        return "odd";
    }
};
console.log(`number 10 is : ${check_even_odd(10)}`);
console.log(`numnber 11 is :${check_even_odd(11)}`);

File: GOLANG_PRACTICAL_EXAM/question_1.go

package main

import ("fmt");


func main() {
	
	// Write a Golang program that checks whether the given number is a prime number or not.
     
	var num int;
	fmt.Println("Enter a number : ");
	fmt.Scan(&num);

	isPrime:=true;

	for i := 2; i < num; i++ {

		if num % i == 0 {
			// fmt.Println(num, "is not a prime number");
			isPrime = false;
			break;
		}
		

	}
    
	fmt.Println("number is prime : ", isPrime);


	

}
